modules/parent-join/src/main/java/org/elasticsearch/join/query/HasChildQueryBuilder.java

```diff
@@ -133,11 +133,11 @@ protected void doWriteTo(StreamOutput out) throws IOException {
      * the maximum number of children that are required to match for the parent to be considered a match.
      */
     public HasChildQueryBuilder minMaxChildren(int minChildren, int maxChildren) {
-        if (minChildren < 0) {
-            throw new IllegalArgumentException("[" + NAME + "] requires non-negative 'min_children' field");
+        if (minChildren <= 0) {
+            throw new IllegalArgumentException("[" + NAME + "] requires greater than 0 'min_children' field");
```

Comment:
Would you mind changing that to simply say "positive"?

Reply:
on it @cbuescher

Reply:
updated @cbuescher